app/scripts/modules/core/src/pipeline/config/stages/managed/publishDeliveryConfigStage.ts

```diff
@@ -0,0 +1,24 @@
+import { Registry } from 'core/registry';
+import { ExecutionDetailsTasks } from 'core/pipeline';
+
+import { PublishDeliveryConfigStageConfig } from './PublishDeliveryConfigStageConfig';
+import { PublishDeliveryConfigExecutionDetails } from './PublishDeliveryConfigExecutionDetails';
+import { IStageOrTriggerBeforeTypeValidationConfig } from 'core/pipeline/config/validation/stageOrTriggerBeforeType.validator';
+
+Registry.pipeline.registerStage({
```

Comment:
Let's wrap this registration call in
```typescript
if (SETTINGS.feature.managedDelivery) {
```
so it doesn't start showing in every Spinnaker installation.